spec/forms/state_file/nc_taxes_owed_form_spec.rb

```diff
@@ -12,9 +12,12 @@
       payment_or_deposit_type: "mail"
     }
   end
-  let(:current_year) { "2024" }
+  let(:current_year) { (MultiTenantService.new(:statefile).current_tax_year + 1).to_s }
```

Comment:
I don't think we should change these dates. If we do we will have to change the test cases every year when current year changes. If this test is failing with updating those config dates we should find some other way to stub those dates in this test

Reply:
You right im about to push up something better